projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -1192,6 +1193,19 @@ export class IgxGridBaseDirective extends DisplayDensityBase implements
     @Output()
     public onCellEdit = new EventEmitter<IGridEditEventArgs>();
 
+    /**
+     * Emitted when cell has been edited and editing has been committed.
```

Comment:
Emitted **after** cell had been edited. After is the important word here IMO

Reply:
done